apps/authentication-portal/src/main/webapp/includes/init-url.jsp

```diff
@@ -33,4 +34,14 @@
         openidServerURL = identityServerEndpointContextParam + "/oidc/logout";
         logincontextURL = identityServerEndpointContextParam + "/logincontext";
     }
+    
+    String tenantDomain;
+    if (IdentityTenantUtil.isTenantQualifiedUrlsEnabled()) {
+        tenantDomain = IdentityTenantUtil.getTenantDomainFromContext();
+        if (StringUtils.isBlank(tenantDomain)) {
+            tenantDomain = request.getParameter("tenantDomain");
+        }
```

Comment:
Should we have this. What if we get rid of the parameter based approach fully at tenanted url mode